.github/workflows/e2e-tests.yml

```diff
@@ -6,6 +6,7 @@ on:
   push:
     branches:
       - trunk
+  workflow_call:
```

Comment:
This makes the workflow reusable - which means that it can be triggered manually, but also can be triggered by other workflows. This is required so we can trigger the tests workflows when we run the release workflow.